src/containers/NFT/NFTHeader/Details.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { CopyToClipboard } from '../../shared/components/CopyToClipboard'
+import './styles.scss'
+import { formatPrice } from '../../shared/utils'
+
+interface Props {
+  data: {
+    NFTId: string
+    ledgerIndex: number | undefined
+    owner: string
+    isBurned: boolean | undefined
+    flags: string[] | undefined
+    transferFee: number
+    issuer: string
+    NFTTaxon: number
+    NFTSequence: number | undefined
+    uri: string | undefined
+    validated: boolean | undefined
+    status: string | undefined
+    warnings: string[] | undefined
+    minted: string | undefined
+    domain: string | undefined
+    emailHash: string | undefined
+  }
+}
+
+const Details = ({ data }: Props) => {
+  const { minted, domain, emailHash, NFTTaxon, uri, transferFee } = data
+  const { t } = useTranslation()
+
+  const formattedFee =
+    transferFee === 0
+      ? `${transferFee}%`
+      : `${formatPrice(transferFee / 1000, {
+          currency: 'none',
+          decimals: 5,
+          padding: 3,
+        })}%`
+
+  const abbrvEmail =
+    emailHash &&
+    (emailHash.length > 20 ? emailHash.slice(0, 20).concat('...') : emailHash)
+  const abbrvURI =
+    uri && (uri.length > 20 ? uri.slice(0, 20).concat('...') : uri)
+
+  return (
+    <table className="token-table">
+      <tbody>
+        {minted && (
+          <tr className="row">
+            <td className="col1">{t('minted')}</td>
+            <td className="col2">{minted}</td>
+          </tr>
+        )}
+        {domain && (
+          <tr className="row">
+            <td className="col1">{t('domain')}</td>
+            <td className="col2">
+              <a
+                href={`https://${domain}`}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                {domain}
```

Comment:
i'm wary about doing this without further investigation. i think for now we should not wrap in an anchor

Reply:
Unresolving because this still has the anchor